Automattic/wp-calypso
# Data Layer: get comment by id

Introduce the ability to request a single comment by id to the data-layer.

To Test:
- run `dispatch({ type: 'COMMENT_REQUEST', siteId: 3584907, commentId: 268320 })`
- You should see it pop up in your state tree under `state.comments.items[siteid-postId]` (along with the network request in devtools)
